app/notifications/validators.py

```diff
@@ -99,17 +103,72 @@ def check_service_over_daily_message_limit(key_type: ApiKeyType, service: Servic
     counter_name="rate_limit.live_service_daily_sms",
     exception=LiveServiceTooManySMSRequestsError,
 )
-def check_service_over_daily_sms_limit_and_warn(key_type: ApiKeyType, service: Service):
-    if current_app.config["FF_SPIKE_SMS_DAILY_LIMIT"] and key_type != KEY_TYPE_TEST and current_app.config["REDIS_ENABLED"]:
-        fragments_sent = fetch_daily_sms_fragment_count(service.id)
-        warn_about_daily_sms_limit(service, fragments_sent)
+def check_sms_daily_limit(service: Service, requested_sms=0):
+    messages_sent = fetch_todays_requested_sms_count(service.id)
+    over_sms_daily_limit = (messages_sent + requested_sms) >= service.sms_daily_limit
+
+    # Send a warning when reaching the daily message limit
+    if not over_sms_daily_limit:
+        return
+
+    current_app.logger.info(
+        f"service {service.id} has been rate limited for daily sms use sent {int(messages_sent)} limit {service.sms_daily_limit}"
```

Comment:
We may want to adjust this text slightly, as this isn't really rate limiting and might confuse us in prod support.

```suggestion
        f"service {service.id} is exceeding their daily sms limit [total sent today: {int(messages_sent)} limit: {service.sms_daily_limit}, attempted send: {requested_sms}"
```

Reply:
Good call!